pkg/operator/controller/controller_dns_configmap_test.go

```diff
@@ -1054,6 +1054,75 @@ foo.com:5353 {
     }
     reload
 }
+`,
+		},
+		{
+			name: "CR with upstreamResolvers containing 1 Network IPv6 NS and policy should return coreFile with 1 upstream defined and that policy",
+			dns: &operatorv1.DNS{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: DefaultDNSController,
+				},
+				Spec: operatorv1.DNSSpec{
+					UpstreamResolvers: operatorv1.UpstreamResolvers{
+						Upstreams: []operatorv1.Upstream{
+							{
+								Type:    operatorv1.NetworkResolverType,
+								Address: "1001:AAAA:bbbb:cCcC::2222",
+								Port:    5353,
```

Comment:
I believe if you omitted the port here, then this unit test would provide coverage for the `else` branch here: https://github.com/openshift/cluster-dns-operator/blob/b402c778bb7b3deda5200b259dce34e61aa96399/pkg/operator/controller/controller_dns_configmap.go#L212.